src/components/ReactGoogleMaps.js

```diff
@@ -241,18 +308,16 @@ export class ReactGoogleMaps extends Component {
   render() {
     console.log(this.props.filterFunction);
     if (this.props.allTaps.length) {
-      var closestTap = closest(this.props.allTaps, {
-        lat: this.state.currlat,
-        lon: this.state.currlon
-      });
+      if((Object.keys(this.state.closestTap).length) === 0){
```

Comment:
Instead of checking for the length of `this.state.closestTap`, should check if location have changed enough to change the closest tap

Reply:
This is OK for now, the functionality to trigger relocating the closestTap can be left to building the closest tap button here: https://github.com/phlask/phlask-map/issues/97